pkgs/applications/misc/immersed-vr/default.nix

```diff
@@ -0,0 +1,23 @@
+{pkgs ? import <nixpkgs> {}, fetchurl, lib }:
+pkgs.appimageTools.wrapType2 {
+  pname = "immersed-vr";
+  version = "9.6";
+
+  src = fetchurl {
+    url = "https://static.immersed.com/dl/Immersed-x86_64.AppImage";
```

Comment:
Could you please get the source from https://web.archive.org?
This url is not versioned and reproducible.

Reply:
Please take a new snapshot and use it.
There is no version information in the URL, but the source is uniquely identified by the snapshot.

Then take a snapshot each time you update and change the version and url.

Reply:
This is. http://web.archive.org/web/20230529010940/https://static.immersed.com/dl/Immersed-x86_64.AppImage

Reply:
No. `20230529010940` is the date, not the version.
You can find many examples of how to use web.archive.org by running grep.

Reply:
see these files.
<details><summary>the result of `rg "web.archive.org" pkgs -l`</summary>

pkgs/README.md
pkgs/tools/text/lv/default.nix
pkgs/tools/text/xml/xml2/default.nix
pkgs/tools/text/gpt2tc/default.nix
pkgs/tools/misc/cmdpack/default.nix
pkgs/tools/misc/xmlbeans/default.nix
pkgs/tools/misc/memtest86-efi/default.nix
pkgs/tools/security/snow/default.nix
pkgs/tools/graphics/stegsolve/default.nix
pkgs/tools/networking/burpsuite/default.nix
pkgs/tools/system/ipmitool/default.nix
pkgs/tools/filesystems/yandex-disk/default.nix
pkgs/tools/compression/offzip/default.nix
pkgs/tools/archivers/unzip/default.nix
pkgs/tools/compression/zdelta/default.nix
pkgs/servers/nominatim/default.nix
pkgs/misc/fastly/default.nix
pkgs/misc/drivers/hplip/default.nix
pkgs/misc/drivers/epson-201401w/default.nix
pkgs/misc/drivers/utsushi/networkscan.nix
pkgs/misc/drivers/epson_201207w/default.nix
pkgs/misc/drivers/epkowa/default.nix
pkgs/misc/drivers/epson-201106w/default.nix
pkgs/misc/drivers/epson-escpr/default.nix
pkgs/misc/cups/drivers/kyocera/default.nix
pkgs/misc/cups/drivers/kyodialog/default.nix
pkgs/misc/drivers/epson-workforce-635-nx625-series/default.nix
pkgs/development/python-modules/sphinx-automodapi/default.nix
pkgs/games/tetrio-desktop/default.nix
pkgs/games/xtris/default.nix
pkgs/games/ue4demos/default.nix
pkgs/games/quake3/content/hires.nix
pkgs/os-specific/linux/intel-ocl/default.nix
pkgs/games/cgoban/default.nix
pkgs/os-specific/darwin/apparency/default.nix
pkgs/os-specific/linux/anbox/postmarketos-image.nix
pkgs/development/tools/frink/default.nix
pkgs/development/libraries/libconfuse/default.nix
pkgs/development/interpreters/lambda-lisp/sources.nix
pkgs/development/libraries/libgda/default.nix
pkgs/development/libraries/libstroke/default.nix
pkgs/development/libraries/soil/default.nix
pkgs/development/compilers/zulu/common.nix
pkgs/development/compilers/gcc/common/libgcc-buildstuff.nix
pkgs/os-specific/darwin/DarwinTools/default.nix
pkgs/data/fonts/quattrocento-sans/default.nix
pkgs/data/fonts/proggyfonts/default.nix
pkgs/data/fonts/times-newer-roman/default.nix
pkgs/data/fonts/vista-fonts-chs/default.nix
pkgs/data/fonts/quattrocento/default.nix
pkgs/data/fonts/linja-pi-pu-lukin/default.nix
pkgs/data/fonts/helvetica-neue-lt-std/default.nix
pkgs/data/fonts/mph-2b-damase/default.nix
pkgs/data/fonts/cooper-hewitt/default.nix
pkgs/data/fonts/babelstone-han/default.nix
pkgs/applications/emulators/citra/default.nix
pkgs/applications/emulators/uae/default.nix
pkgs/applications/emulators/darcnes/default.nix
pkgs/applications/version-management/p4d/default.nix
pkgs/applications/version-management/rcshist/default.nix
pkgs/applications/video/flirc/default.nix
pkgs/applications/version-management/p4/default.nix
pkgs/applications/version-management/p4v/default.nix
pkgs/applications/video/filebot/default.nix
pkgs/applications/virtualization/driver/win-pvdrivers/default.nix
pkgs/applications/science/math/geogebra/geogebra6.nix
pkgs/applications/science/math/geogebra/default.nix
pkgs/applications/science/biology/aragorn/default.nix
pkgs/applications/office/banana-accounting/default.nix
pkgs/applications/office/clockify/default.nix
pkgs/applications/misc/slic3r/default.nix
pkgs/applications/misc/parsec/bin.nix
pkgs/applications/misc/kjv/default.nix
pkgs/applications/networking/tetrd/default.nix
pkgs/applications/networking/mailreaders/bluemail/default.nix
pkgs/applications/networking/instant-messengers/skypeforlinux/default.nix
pkgs/applications/networking/instant-messengers/vk-messenger/default.nix
pkgs/applications/networking/instant-messengers/threema-desktop/default.nix
pkgs/applications/networking/instant-messengers/viber/default.nix
pkgs/applications/kde/koko.nix
pkgs/applications/graphics/odafileconverter/default.nix
pkgs/applications/graphics/leocad/default.nix
pkgs/applications/audio/ardour/default.nix
pkgs/applications/audio/renoise/default.nix
pkgs/applications/audio/spotify/darwin.nix
pkgs/applications/audio/losslessaudiochecker/default.nix
pkgs/applications/audio/soundsource/default.nix
pkgs/applications/audio/soundwireserver/default.nix
pkgs/applications/audio/bespokesynth/default.nix
pkgs/applications/audio/oxefmsynth/default.nix
pkgs/desktops/cinnamon/mint-artwork/default.nix
</details>